### DISCUSSION TITLE
rss订阅生成错误

### DISCUSSION THREAD

**Discussion by danbai225** [halo-dev/halo#2198] [Q&A] [answered]
### 检查清单

- [X] 已经在 [GitHub Issues](https://github.com/halo-dev/halo/issues) 中搜索过该问题
- [X] 已经在[论坛](https://bbs.halo.run/)中搜索过该问题
- [X] 已经查阅过[文档](https://docs.halo.run/)

### 是什么版本出现了此问题？

1.5.3

### 使用的什么数据库？

H2

### 使用的哪种方式部署？

Docker

### 在线站点地址

https://p00q.cn/rss.xml

### 发生了什么？

最近打开rss后发现有报错，可能跟我最近写的文章编码有关系，但我没找到。
日志里没有报错
`
This page contains the following errors:
error on line 31 at column 34: Input is not proper UTF-8, indicate encoding !
Bytes: 0x08 0x7B 0xE6 0xB5
Below is a rendering of the page up to the first error.
`

**Comment by guqing** [comment, marked as answer]
由于你的文章中出现了不完整的 utf-8 字符导致浏览器无法显示 rss 的 xml，通过如下图片可以看出 halo 正确反回了 rss.xml，因此这并不是 halo 的问题
![image](https://user-images.githubusercontent.com/38999863/176185743-dafa8d07-4cd5-4a48-bd51-d44e9fa0935f.png)

你只需要把如下这篇文章中的红点字符去掉即可
![image](https://user-images.githubusercontent.com/38999863/176185369-dd6ce5ef-de6f-40b5-aa1b-9a0957ef0fce.png)
